test/integration/fleetscope/fleetscope_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fleetscope
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+)
+
+func TestFleetscope(t *testing.T) {
+
+	for _, envName := range []string{
+		"development",
+		"non-production",
+		"production",
+	} {
+		envName := envName
+		t.Run(envName, func(t *testing.T) {
+			multitenant := tft.NewTFBlueprintTest(t,
+				tft.WithTFDir(fmt.Sprintf("../../../2-multitenant/envs/%s", envName)),
+			)
+
+			vars := map[string]interface{}{
+				"project_id":             multitenant.GetTFSetupStringOutput("project_id"),
```

Comment:
```suggestion
				"project_id":             multitenant.GetStringOutput("project_id"),
```

Reply:
Bad copy/paste in my original example, these shouldn't be the "TFSetup" variant

Reply:
Pending release of https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/2159

Reply:
Pending update of module in this repo: https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/21

Reply:
updated the module with your fix commit